springfield/cms/templates/components/banner.html

```diff
@@ -25,12 +25,11 @@
 
   <div class="fl-banner-layout">
     {% if contents.media or qr_code %}
-      <div class="fl-banner-media" aria-hidden="true" {% if media_url and qr_code %} style="background-image: url('{{ media_url }}');" {% endif %}>
+      <div class="fl-banner-media {% if qr_code %} has-qr-code{% endif %} {% if qr_code and not contents.media %} has-large-qr-code{% endif %}" aria-hidden="true">
         {% if qr_code %}
           <div class="fl-banner-qr" role="img">{{ contents.qr }}</div>
```

Comment:
We have some existing translations for "Scan this QR code" on the evergreen WNP

https://github.com/mozilla-l10n/www-firefox-l10n/blob/164f29690690aeb56b852ae46f7f2dc34b04dee9/fr/firefox/whatsnew/evergreen.ftl#L16

https://github.com/mozilla-l10n/www-firefox-l10n/blob/164f29690690aeb56b852ae46f7f2dc34b04dee9/de/firefox/whatsnew/evergreen.ftl#L16